lib/attache/download.rb

```diff
@@ -48,7 +48,7 @@ def _call(env, config)
           nil
         end
 
-        unless file
+        unless file && file.try(:size).to_i > 0
```

Comment:
I did not see Active Support listed in the [dependencies](https://github.com/choonkeat/attache/blob/master/attache.gemspec#L22-L34). We don't have `try` method available?